### PR TITLE
Feature: Maintain only server hooks :heart:

### PR DESCRIPTION
**Pull request by gabyx**
To issue #63

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,6 +255,9 @@ git hooks config accept trusted
 git hooks config disable update
 ```
 
+Bare repositories will always
```
Comment (rycus86) [inline README.md]: Unfinished sentence?

File: install.sh [hunk 2]
```diff
@@ -3819,6 +3820,16 @@ setup_hook_templates() {
...
+        INSTALL_ONLY_SERVER_HOOKS="yes"
+    fi
+
+    if [ "$INSTALL_ONLY_SERVER_HOOKS" = "yes" ]; then
+        HOOK_NAMES="$MANAGED_SERVER_HOOK_NAMES"
+    else
+        HOOK_NAMES="$MANAGED_HOOK_NAMES"
+    fi
+
     for HOOK in $MANAGED_HOOK_NAMES; do
```
Comment (rycus86) [inline install.sh]: Did you need to use `$HOOK_NAMES` perhaps here?

**Comment by rycus86** [comment]
Could you please add a quick test for this as well to check that the expected hooks are getting installed? Thanks!

File: install.sh (hunk 2 above)
Comment (gabyx) [reply install.sh]: Ups.

**Comment by gabyx** [comment]
Review changes. Added some more hooks. Added a test. Will fix tonight if it does not work...

File: install.sh [hunk 3]
```diff
@@ -3455,19 +3456,21 @@ parse_command_line_arguments() {
...
-
-            # Using core.hooksPath implies it applies to all repo's
-            if [ "$SINGLE_REPO_INSTALL" = "yes" ]; then
-                echo "! Cannot use --single and --use-core-hookspath together" >&2
-                exit 1
-            fi
         fi
         prev_p="$p"
     done
+
```
Comment (gabyx) [inline install.sh]: Better validate everything after parsing...

File: uninstall.sh [hunk 4]
```diff
@@ -396,29 +396,28 @@ uninstall() {
...
     git config --global --unset githooks.autoupdate.enabled
     git config --global --unset githooks.autoupdate.lastrun
     git config --global --unset githooks.previous.searchdir
     git config --global --unset githooks.disable
     git config --global --unset githooks.installDir
+    git config --global --unset githooks.pathForUseCoreHooksPath
+    git config --global --unset githooks.useCoreHooksPath
     git config --global --unset alias.hooks
 
-    if using_hooks_path; then
```
Comment (gabyx) [inline uninstall.sh]: I moved the removing of `githooks.` at a central place. Always clean up everything, **no if ... else ...** Its uninstall.

File: uninstall.sh [hunk 5]
```diff
@@ -396,29 +396,28 @@ uninstall() {
     # Uninstall all cli
     uninstall_cli
 
+    if using_hooks_path; then
```
Comment (gabyx) [inline uninstall.sh]: Same code. Without the `githooks` stuff

File: README.md [hunk 6]
```diff
@@ -202,13 +202,13 @@ $ sh -c "$(curl -fsSL https://r.viktoradam.net/githooks)" -- --use-core-hookspat
 By default the script will install the hooks into `~/.githooks/templates/`, optionally, you can also pass the path to which you want to install the centralized hooks by appending `<path>` to the command above, for example:
 
 ```shell
-$ sh -c "$(curl -fsSL https://r.viktoradam.net/githooks)" -- --use-core-hookspath /home/public/.githooks
+$ sh -c "$(curl -fsSL https://r.viktoradam.net/githooks)" -- --use-core-hookspath
```
Comment (gabyx) [inline README.md]: Wrong doc.

**Comment by gabyx** [comment]
Added a test. Everything works. Read to merge :-) Thx

File: install.sh (hunk 2 above)
Comment (rycus86) [reply install.sh]: Did you need to change this here?
If not, why the `HOOK_NAMES=...` above?

Comment (gabyx) [reply install.sh]: Typo. Thx

Comment (gabyx) [reply install.sh]: Hm.. why did the test pass ;_)

**Comment by gabyx** [comment]
Haha, the test was wrong, hui...

**Comment by gabyx** [comment]
Ok, now its good to go, The test was wrong, thats why I did not see the missing $HOOK_NAMES...

**Comment by rycus86** [comment]
OK, this looks good, just resolve the conflicts please.

**Pull request merged**